packages/story-editor/src/components/colorPicker/test/colorPicker/moveStopByPointer.js

```diff
@@ -49,17 +49,17 @@ describe('<ColorPicker /> when moving a stop with a pointer device', () => {
       clientX: 0,
     });
 
-    // Move to the 20% mark
+    // Move to the 30% mark
```

Comment:
Curious -- why the change?

Reply:
Rounding differences due to the decreased width of the entire bar - the previous 20% rounded to 21% because 🪄 .